src/aap_eda/api/serializers/utils.py

```diff
@@ -140,3 +140,85 @@ def inputs_to_store(inputs: dict, old_inputs_str: str = None) -> str:
 
 def inputs_from_store(inputs: str) -> dict:
     return yaml.safe_load(inputs)
+
+
+def validate_inputs(schema: dict, inputs: dict) -> dict:
```

Comment:
@bzwei can you add please some test data around this?

Reply:
@bzwei Also lets move this to a different file maybe credentials/utils.py or credentials_utils.py

Reply:
moved to aap_eda.core.utils.credential_utils.py